lib/msal-angular/src/IMsalService.ts

```diff
@@ -4,26 +4,28 @@
  */
 
 import {
-    EndSessionRequest,
-    AuthorizationUrlRequest,
-    AuthenticationResult,
-    PopupRequest,
-    RedirectRequest,
-    SilentRequest,
-    Logger
-} from "@azure/msal-browser";
-import { Observable } from "rxjs";
+  EndSessionRequest,
+  AuthorizationUrlRequest,
+  AuthenticationResult,
+  PopupRequest,
+  RedirectRequest,
+  SilentRequest,
+  Logger,
+} from '@azure/msal-browser';
```

Comment:
Yes.  I didn't override any of the recommended settings from Prettier, mostly to avoid ever having to discuss the pros/cons of quote types in the future ;-)

Reply:
Here's the prettier teams opinion on options: https://prettier.io/docs/en/option-philosophy.html and Dan Abramov's issue referenced in that article: https://github.com/prettier/prettier/issues/40

Reply:
I mentioned in teams that this is likely worth a discussion.  I see where you're coming from here, but let's discuss the tradeoffs and any concerns if folks have any.

Reply:
Sounds good, my concern is just that keeping dependencies up to date is a constant struggle and pinning dependencies makes that harder. I'm ok to try this and re-evaluate later if it becomes burdensome.